deployment/stacks/ui.ts

```diff
@@ -95,6 +93,20 @@ export function createUiComponents(props: CreateUiComponentsProps) {
     }
   );
 
+  // new cloudfront.Distribution(
```

Comment:
This is the simplified version of the distribution that will initially be used for import. Once imported we'll update it to bring it back into the fully featured setup Jon got working for cdk.